### PR TITLE
[T4A5][T17-B1]Chia Zhe Min

### PR DESCRIPTION
**Pull request by zhems**

### CODE REVIEW

**Comment by zhems** [comment]
Ready for review

File: src/seedu/addressbook/data/AddressBook.java [hunk 1]
```diff
@@ -22,6 +21,8 @@
 
     private final UniquePersonList allPersons;
     private final UniqueTagList allTags; // can contain tags not attached to any person
+    
+    private final List<Tagging> taggingHistory = new ArrayList<>(); //will contain all history of all tagging
```
Comment (JoelT-92) [inline src/seedu/addressbook/data/AddressBook.java:25]: Nicely done. Remember that this is a composition relationship.

File: src/seedu/addressbook/data/tag/Tagging.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import java.util.*;
+
+public class Tagging {
+    
+    public static String toPrint = "";
+    public String tagging;
+    
+    public Tagging(String name, Tag tag, boolean add) {
+        if(add) {
+            tagging = "+ " + name + " [" + tag.toString() +"]";
```
Comment (JoelT-92) [inline src/seedu/addressbook/data/tag/Tagging.java:16]: Magic strings used for formatting. You may wish to refactor this part.

**Comment by JoelT-92** [comment]
@zhems Some comments added. Please close the PR after reading comments. Good job! Apparently some of the classmates from the Thursday 1pm slot referenced your code! Be proud!

**Comment by zhems** [comment]
@JoelT-92 haha saw it thanks!

**Pull request closed without merging**